### PR TITLE
Hotfix/cf changed permissions

### PR DESCRIPTION
**Pull request by andrew-jameson**
## Summary of Changes
Includes workaround provided in CF slack channel

### CODE REVIEW

**Review by raftmsohani — APPROVED** [review]
LGTM!

File: .circleci/infrastructure/commands.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@
             apk add jq
             apk add curl
             # TODO: Add Signature check
-            curl -L "https://packages.cloudfoundry.org/stable?release=linux64-binary&version=v7&source=github" | tar -zx
+            curl -A "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36" -L  "https://packages.cloudfoundry.org/stable?release=linux64-binary&version=v7&source=github" | tar -zx
```
Comment (andrew-jameson) [inline .circleci/infrastructure/commands.yml]:
```suggestion
            curl -A "user-agent" -L  "https://packages.cloudfoundry.org/stable?release=linux64-binary&version=v7&source=github" | tar -zx
```

File: .circleci/infrastructure/commands.yml [hunk 2]
```diff
@@ -87,7 +87,7 @@
             sudo apt install jq
             sudo apt install curl
             # TODO: Add Signature check
-            curl -L "https://packages.cloudfoundry.org/stable?release=linux64-binary&version=v7&source=github" | tar -zx
+            curl -A "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36" -L "https://packages.cloudfoundry.org/stable?release=linux64-binary&version=v7&source=github" | tar -zx
```
Comment (andrew-jameson) [inline .circleci/infrastructure/commands.yml]:
```suggestion
            curl -A "user-agent" -L "https://packages.cloudfoundry.org/stable?release=linux64-binary&version=v7&source=github" | tar -zx
```

File: scripts/cf-check.sh [hunk 3]
```diff
@@ -7,7 +7,7 @@ else
     apt-get update
     apt-get install wget gnupg2 apt-transport-https
 
-    wget -q -O - https://packages.cloudfoundry.org/debian/cli.cloudfoundry.org.key | sudo apt-key add -
+    wget -U "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36" -q -O https://packages.cloudfoundry.org/debian/cli.cloudfoundry.org.key | sudo apt-key add -
```
Comment (andrew-jameson) [inline scripts/cf-check.sh]:
```suggestion
    wget -U "user-agent" -q -O https://packages.cloudfoundry.org/debian/cli.cloudfoundry.org.key | sudo apt-key add -
```

**Review by ADPennington — APPROVED** [review]

**Comment by andrew-jameson** [comment]
OBE, they fixed their anonymous access

**Pull request closed without merging**